roles/prometheus/tasks/main.yml

```diff
@@ -39,3 +39,8 @@
     - restart consul
   tags:
     - prometheus
```

Comment:
Does this mean we are running this db container on every host on the cluster? is that necessary?